src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+
+  </Side>
+  </SideBySide>
+
+# Live archives [#live-archives]
+<SideBySide>
+  <Side>
+
+  Live archives store historical logs with other logs and telemetry data in the New Relic Database. This allows instant access to old logs and stops the need to reload, re-index logs, or move data between many locations or tiers for analysis.
+
+  Learn more about live archives [here](/docs/logs/get-started/log-live-archives).
+
+  </Side>
+  <Side>
+
+  <img
+    title="New Relic Live archives"
+    alt="New Relic Live archives"
+    src={NRLiveArchivesMain}
+  />
+
+  </Side>
+  </SideBySide>
+
+# Logs [#logs]
+<SideBySide>
+  <Side>
+
+  <img
+    title="New Relic Live archives"
+    alt="New Relic Live archives"
+    src={NRLogsMain}
+  />
+
+  </Side>
+  <Side>
+
+  New Relic can help you manage large volumes of your log data. We provide tools to ingest only valuable logs to cut costs. We offer a unified UI to connect your <InlinePopover type="logs" /> to your services. We also offer tools that can help you sort through the clutter. 
+
+  Learn more about logs [here](/docs/logs/get-started/get-started-log-management/).
+
+  </Side>
+  </SideBySide>
+
+# Network [#network]
+<SideBySide>
+  <Side>
+
+  With <InlinePopover type="networkMonitoring" />, you can analyze how well your routers, switches, and other networking devices perform. It will also build a visual network topology, letting you find performance problems in your environment. 
+  
+  Learn more about network [here](/docs/network-performance-monitoring/get-started/npm-introduction/).
+
+  </Side>
+  <Side>
+
+<img
+  title="New Relic network"
+  alt="New Relic network"
+  src={NRNetworkMain}
+/>
+
+  </Side>
+  </SideBySide>
+
+# Mobile [#mobile]
+<SideBySide>
+  <Side>
+<img
+  title="New Relic mobile"
+  alt="New Relic mobile"
+  src={NRMobileMain}
+/>
+
+  </Side>
+  <Side>
+
+  The <InlinePopover type="mobile" /> capability sends telemetry directly from the end user's mobile app to New Relic. This allows us to analyze the app's performance on the user's device. This capability helps with the performance and crash troubleshooting of your Android, iOS, or hybrid mobile apps. Use mobile monitoring to improve your app's user experience. Or, study HTTP and network performance to work better with your backend teams.
+
+  Learn more about mobile [here](/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-mobile-monitoring/).
+
+  </Side>
+  </SideBySide>
+
+# OpenTelemetry [#opentelemetry]
+<SideBySide>
+  <Side>
+
+  OpenTelemetry helps collect data from your applications and hosts, sending it to New Relic. New Relic's platform lets you analyze the data and fix any application problems. This toolkit simplifies the process of monitoring and troubleshooting your software. By utilizing OpenTelemetry, you can enhance the performance and reliability of your applications.
```

Comment:
@rhetoric101 Is OpenTelemetry an integration? I'm a little hazy on the details here, but the way I understand it is that New Relilc has some kind of compatibility with OTel so that OTel customers can forward that data into New Relic. 

i think this section might benefit from describing why would someone might OTel _and_ New Relic? Or maybe telling a story about how using New Relic enhances OTel monitoring

Reply:
I'm going to wait for what Rob has to say here.